doc/source/performance/procfs-plugin.rst

```diff
@@ -0,0 +1,139 @@
+.. procfs-plugin:
+
+======================================================================
+The ProcFS plugin
+======================================================================
+
+This feature is **tech preview** quality.
+
+Implemented in :rn:`8.0.25-15`, the ProcFS plugin provides access to the Linux performance counters by running SQL queries against a Percona Server for MySQL 8.0 or another MySQL 8.0-compatible variant. 
```

Comment:
According to some recent feedback from @georgelorch , we do not guarantee plugin compatibility with other MySQL variants other than Percona Server (and PXC). So I'd suggest to remove the "and another MySQL 8.0-compatible variant" here.

Reply:
Done.

Reply:
Done.